### PR TITLE
Import scripts for Decennial Census

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: scripts/us_census/decennial/process.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+_DATAF_LOGRECNO_COL = 4
+
+# Delimiter for data file varies across years.
+_DATAF_DELIM_CHAR = {
+    '2010': ',',
+    '2020': '|',
+}
+
+# Table name as it appears in Data Dictionary Reference:
+# https://user-images.githubusercontent.com/4375037/130696166-da194721-c8e1-484e-8722-09202d3fe5c3.png
```
Comment (beets) [inline scripts/us_census/decennial/process.py:94]: maybe also add the source for these images once in the file header? unfortunately, unlike screenshot tool there isn't a ready link generated.

**Review by beets — APPROVED** [review]
don't forget to add __init__.py so tests will run with ./run_tests.sh

File: scripts/us_census/decennial/process.py (hunk 1 above)
Comment (pradh) [reply scripts/us_census/decennial/process.py:99]: Added a comment noting the source docs.

**Comment by pradh** [comment]
> don't forget to add **init**.py so tests will run with ./run_tests.sh

Thanks for the reminder (I actually never realized). I added a line to our top-level README, but its probably worth some kind of presubmit check, as some recent scripts, like s2201, are missing them too...

I tried `./run_tests.sh -a` and it failed badly complaining about some numpy thing (below). I'm assuming its just my env since cloudbuild runs fine.  maybe we should try out your suggestion to decouple requirements.txt https://groups.google.com/a/google.com/g/datcom-core/c/WBxk1eEinvY/m/NC6rj0utAQAJ ?

```
      compile options: '-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -Inumpy/core/include -Ibuild/src.linux-x86_64-3.9/numpy/core/include/numpy -Inump
y/core/src/private -Inumpy/core/src -Inumpy/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inumpy/core/src/umath -Inumpy/core/src/npysort -I/usr/local/google/home
/shanth/work/git/data/.env/include -I/usr/include/python3.9 -Ibuild/src.linux-x86_64-3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9/numpy/core/src/npymath -Ibuild/sr
c.linux-x86_64-3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9/numpy/core/src/npymath -Ibuild/src.linux-x86_64-3.9/numpy/core/src/private -Ibuild/src.linux-x86_64-3.9
/numpy/core/src/npymath -c'                                                                                                                                                   
```

**Comment by pradh** [comment]
Thanks for the review!

**Comment by beets** [comment]
Yeah, a presubmit check and env setup script would be good to add. Sorry it
fell off my plate.. :-/

The test script should use the environment though, and install requirements
before running the tests. Did any of the tests run, or was this an error
that came from installing requirements?

**Comment by pradh** [comment]
no worries, fell off mine too :)

Yes, the errors come just from installing requirements...


   command: /usr/local/google/home/shanth/work/git/data/.env/bin/python3
/usr/local/google/home/shanth/work/git/data/.env/lib/python3.9/site-packages/pip
install --ignore-installed --no-user --prefix
/tmp/pip-build-env-0dntgr33/overlay --no-warn-script-location --no-binary
:none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel
'Cython>=0.29.13' 'numpy==1.13.3; python_version=='"'"'3.6'"'"' and
platform_system!='"'"'AIX'"'"'' 'numpy==1.14.5;
python_version>='"'"'3.7'"'"' and platform_system!='"'"'AIX'"'"''
'numpy==1.16.0; python_version=='"'"'3.6'"'"' and
platform_system=='"'"'AIX'"'"'' 'numpy==1.16.0;
python_version>='"'"'3.7'"'"' and platform_system=='"'"'AIX'"'"''
       cwd: None

  Complete output (4715 lines):

  Ignoring numpy: markers 'python_version == "3.6" and platform_system !=
"AIX"' don't match your environment
  Ignoring numpy: markers 'python_version == "3.6" and platform_system ==
"AIX"' don't match your environment
  Ignoring numpy: markers 'python_version >= "3.7" and platform_system ==
"AIX"' don't match your environment

**Comment by pradh** [comment]
Thanks for the review!

**Pull request merged**